grype/presenter/format.go

```diff
@@ -11,6 +11,8 @@ const (
 	cycloneDXFormat format = "cyclonedx"
 	sarifFormat     format = "sarif"
 	templateFormat  format = "template"
+	embeddedVEXJSON format = "embedded-cyclonedx-vex-json"
+	embeddedVEXXML  format = "embedded-cyclonedx-vex-xml"
```

Comment:
these are accurate, though pretty verbose. I'll give a little more thought to something that is shorter / friendlier to input.

Reply:
I don't have any suggestions for how these could be shorter. @wagoodman did you still have thoughts?